packages/openapi-v3/test/unit/decorators/param/param-header.decorator.unit.ts

```diff
@@ -13,18 +13,14 @@ describe('Routing metadata for parameters', () => {
         @get('/greet')
         greet(@param.header.string('name') name: string) {}
       }
-
-      const actualSpec = getControllerSpec(MyController);
-
-      expect(actualSpec.paths['/greet']['get'].parameters).to.eql([
-        {
-          name: 'name',
-          in: 'header',
-          schema: {
-            type: 'string',
-          },
+      const expectedParamSpec = {
+        name: 'name',
+        in: 'header',
+        schema: {
+          type: 'string',
         },
-      ]);
+      };
+      runTest(MyController, expectedParamSpec);
```

Comment:
It's a minor thing but instead of `runTest`, I'd prefer a name with `expect` in them. `expectSpecToBeEqual` or something along that line

Reply:
The name `expectSpecToBeEqual` sounds better 👍